sourmash/commands.py

```diff
@@ -338,7 +338,8 @@ def index(args):
     if args.append:
         tree = load_sbt_index(args.sbt_name)
     else:
-        tree = create_sbt_index(args.bf_size, n_children=args.n_children)
+        tree = create_sbt_index(args.bf_size, n_children=args.n_children,
+                                not_localized=args.not_localized)
```

Comment:
This can be decided inside `create_sbt_index` by checking the `n_children` value, so I think it's fine to revert.

```suggestion
        tree = create_sbt_index(args.bf_size, n_children=args.n_children)
```